skimage/data/_detect.py

```diff
@@ -0,0 +1,16 @@
+import os as _os
+data_dir = _os.path.abspath(_os.path.dirname(__file__))
+
+
+def frontal_face_cascade_xml():
```

Comment:
The name is not very clear, I would prefer ``frontal_face_cascade_filename`` but it might be a matter of taste only, feel free to ignore this particular remark.